README.md

```diff
@@ -136,28 +136,28 @@ Let's demystify the counter apps and follow their integration step by step. Ther
 
 #### `modules/asset`
 
-The module is inherited from the original Starter Kit Essentials repository. It provides the original CSS used in the starter kit. For the purposes of this demo, we added the Tailwind CSS entrypoint (see `ui/src/index.js`) and a common `svg` asset files (`ui/svg`) referenced by the Counter App UI components.
+The module is inherited from the original Starter Kit Essentials repository and is simply a convenience for organizing some of our assets. It provides the original CSS used in the starter kit. For the purposes of this demo, we added the Tailwind CSS entrypoint (see `ui/src/index.js`) and a common `svg` asset files (`ui/svg`) referenced by the Counter App UI components.
 
 #### `modules/counter`
 
 This module contributes the back-end logic required to save the counter value per App Counter widget (on counter button click). The module provides: 
-- A simple API endpoint to save the counter value in the MongoDB database per widget instance.
-- A method to get the counter value per widget instance, used in the async server component to pass that value as a prop to the front-end app.
-- Nunjucks helpers and component to serialize server-side data and send it to the front-end app via `data-*` attributes.
+- A simple API endpoint using the Apostrophe `apiRoutes(self)` configuration method to save the counter value in the MongoDB database per widget instance.
+- A method, `getWidgetCounter(id)`, to get the counter value per widget instance, used in the async server component to pass that value as a prop to the front-end app.
+- A Nunjucks helper filter (`toAttributeValue(obj)`) and component. The component is defined in the `counterApp()` component method and the template is located in the `views` folder. In ApostropheCMS, components act much like they do in other frameworks, allowing you to add specific functionality to any of your templates. In this case, we are serializing server-side data and sending it to the front-end app via `data-*` attributes.
```

Comment:
"and a server component". I'm trying to be consistent with the term `component` which the modern dev understands as a Framework component, but in this case I've missed that.

Reply:
Changed.